buidler.config.js

```diff
@@ -0,0 +1,32 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+
+usePlugin('@nomiclabs/buidler-truffle5')
+usePlugin('@nomiclabs/buidler-web3')
+usePlugin('@aragon/buidler-aragon')
+
+module.exports = {
+  defaultNetwork: 'localhost',
+  networks: {
+    localhost: {
+      url: 'http://localhost:8545',
+      accounts: [
+        '0xa8a54b2d8197bc0b19bb8a084031be71835580a01e70a45a13babd16c9bc1563',
```

Comment:
Oh interesting, does this select the accounts that are unlocked?

Reply:
Yes, since atm we're using aragen separately, we need to specify which accounts are available. When we use Buidler EVM, this shouldn't be necessary anymore.